vmdk_plugin/Makefile

```diff
@@ -95,10 +98,18 @@ VMDKOPS_TEST_MODULE := vmdkops
 VMDKOPS_MODULE_SRC = $(VMDKOPS_MODULE)/*.go $(VMCI_SRC)
 
 # All sources. We rebuild if anything changes here
-SRC = main.go log_formatter.go utils/refcount/refcnt.go \
+SRC = original_plugin/main.go original_plugin/constants.go \
```

Comment:
"original" is not semantically clear name. We can call it "block_device" or something along these lines, but not "original"